hyundai_kia_connect_api/KiaUvoApiCA.py

```diff
@@ -344,26 +348,30 @@ def lock_action(self, token: Token, vehicle: Vehicle, action) -> str:
         )
         response_headers = response.headers
         response = response.json()
+        self.last_action_xid = response_headers["transactionId"]
+        self.last_action_pin_auth = headers["pAuth"]
 
         _LOGGER.debug(f"{DOMAIN} - Received lock_action response")
-        return response_headers["transactionId"]
 
     def start_climate(
         self, token: Token, vehicle: Vehicle, options: ClimateRequestOptions
     ) -> str:
-        url = self.API_URL + "rmtstrt"
+        if vehicle.engine_type == ENGINE_TYPES.EV:
+            url = self.API_URL + "evc/rfon"
+
+        else: 
+            url = self.API_URL + "rmtstrt"
         headers = self.API_HEADERS
         headers["accessToken"] = token.access_token
         headers["vehicleId"] = vehicle.id
         headers["pAuth"] = self._get_pin_token(token)
-
-        if vehicle.year < 2020:
+        if vehicle.year > 2020:
```

Comment:
I would move `2020` into a static string to the top of this file and put a comment there :)